scripts/userstyles.yml

```
@@ -871,7 +872,7 @@ userstyles:
     icon: tldraw
     color: text
     note: This theme only changes how the colors appear on the tldraw canvas. Exported graphics **will not be Catppuccin-themed**.
-    current-maintainers: [*GenShibe, *uncenter]
+    current-maintainers: [*NekoDrone]
```

https://userstyles.catppuccin.com/contributing/userstylesyml/#removing-yourself-as-a-maintainer
```suggestion
    current-maintainers: [*NekoDrone]
    past-maintainers: [*GenShibe, *uncenter]
```